cpp/examples/ode_secir_ageres.cpp

```diff
@@ -73,38 +73,62 @@ int main()
         model.populations.set_difference_from_group_total<mio::AgeGroup>({i, mio::osecir::InfectionState::Susceptible},
                                                                          fact * nb_total_t0);
 
-        params.get<mio::osecir::TransmissionProbabilityOnContact>()[i] = 0.05;
-        params.get<mio::osecir::RelativeTransmissionNoSymptoms>()[i]   = 0.67;
-        params.get<mio::osecir::RecoveredPerInfectedNoSymptoms>()[i]   = 0.09;
-        params.get<mio::osecir::RiskOfInfectionFromSymptomatic>()[i]   = 0.25;
-        params.get<mio::osecir::SeverePerInfectedSymptoms>()[i]        = 0.2;
-        params.get<mio::osecir::CriticalPerSevere>()[i]                = 0.25;
-        params.get<mio::osecir::DeathsPerCritical>()[i]                = 0.3;
+        params.get<mio::osecir::TransmissionProbabilityOnContact>()[i]  = 0.05;
+        params.get<mio::osecir::RelativeTransmissionNoSymptoms>()[i]    = 0.7;
+        params.get<mio::osecir::RecoveredPerInfectedNoSymptoms>()[i]    = 0.09;
+        params.get<mio::osecir::RiskOfInfectionFromSymptomatic>()[i]    = 0.25;
+        params.get<mio::osecir::MaxRiskOfInfectionFromSymptomatic>()[i] = 0.45;
+        params.get<mio::osecir::SeverePerInfectedSymptoms>()[i]         = 0.2;
+        params.get<mio::osecir::CriticalPerSevere>()[i]                 = 0.25;
+        params.get<mio::osecir::DeathsPerCritical>()[i]                 = 0.3;
     }
 
+    params.apply_constraints();
```

Comment:
```suggestion
    model.apply_constraints();
```
we should always try to call the apply_constraints() function from the model level because then we also check the population. See
https://github.com/SciCompMod/memilio/blob/1fa9e20ebba466cc7b12f010a29249c4fa5e4722/cpp/memilio/compartments/compartmentalmodel.h#L137